packages/synthetics-sdk-broken-links/test/unit/broken_links.spec.ts

```diff
@@ -55,14 +63,11 @@ describe('GCM Synthetics Broken Links Core Functionality', async () => {
 
     it('should pass on the first attempt', async () => {
```

Comment:
A few interactions that I think still need to be tested - target url doesnt load, target url returns content that isnt html (like it returns a json object or something).

Reply:
I think the target_url not loading is capture in the Timeout test. I added a test that visits a custom .json file instead of .html

Reply:
Sorry I should have been more specific, a 400 or 500 level error where no content is actually returned other than an error message. But yes, the timeout test is also helpful :)

Reply:
Ah ok, yes. thanks!